tiptapy/__init__.py

```diff
@@ -58,15 +58,19 @@ class Image(BaseNode):
     def inner_render(self, node) -> str:
         special_attrs_map = {'caption': 'figcaption'}
         attrs = node.get("attrs", {})
-        attrs_s = " ".join(f'{k}="{v}"'
-                           for k, v in attrs.items()
-                           if k not in special_attrs_map
-                           )
+        attrs_s = " ".join(
+            f'{k}="{v}"'
+            for k, v in attrs.items()
+            if k not in special_attrs_map and
+            len(attrs[k]) != 0
```

Comment:
@Pradhvan we don't need to check length for empty string.

Reply:
```
attrs_s = " ".join(
            f'{k}="{v}"'
            for k, v in attrs.items()
            if k not in special_attrs_map and
            len(attrs[k]) != 0
        )
"""
attrs_s gives this -> src="https://placekitten.com/200/301"
if I remove the len check it gives -> src="https://placekitten.com/200/301" alt=""
"""
```
`attrs_s` gives the string response of the node and ` len(attrs[k]) != 0` just checks if the `alt` tags should be added or not if in the given JSON it has an empty string. 

@gauravr  I did not want to do something like `attrs[k] == ' '` that's why I added `len(attrs[k]) != 0`

Reply:
you can use
```
attrs_s = " ".join(
            f'{k}="{v}"'
            for k, v in attrs.items()
            if k not in special_attrs_map and v
        )
```

Reply:
@gauravr yes, this looks more readable and better.